cmd/amp/account.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Interactive
+	loginCmd = &cobra.Command{
+		Use:   "login",
+		Short: "log in to amp",
+	}
+
+	accountCmd = &cobra.Command{
+		Use:   "account",
+		Short: "account operations",
+	}
+	// The rest of the commands are subcommands
+
+	// Interactive
+	signUpCmd = &cobra.Command{
+		Use:   "signup",
+		Short: "Create a new account and login",
+	}
+
+	// Interactive
+	verifyCmd = &cobra.Command{
+		Use:   "verify [CODE]",
+		Short: "verify email using code",
+	}
+
+	assumeRoleCmd = &cobra.Command{
+		Use:   "assume [ORGANIZATION] [TEAM]",
+		Short: "assume role as organation or team",
+	}
+
+	createOrganizationCmd = &cobra.Command{
+		Use:   "create organization [NAME] [EMAIL]",
+		Short: "Create an organization",
+	}
+
+	listUsersCmd = &cobra.Command{
+		Use:   "list users [ORGANIZATION] [TEAM]",
+		Short: "list users, optionally filter by organation and team",
+	}
+
+	listOrganizationsCmd = &cobra.Command{
+		Use:   "list organizations",
+		Short: "list organizations",
+	}
+
+	listTeamsCmd = &cobra.Command{
+		Use:   "list teams [ORGANIZATION]",
+		Short: "list teams by organization",
+	}
+
+	listPermissionsCmd = &cobra.Command{
+		Use:   "list permissions [ORGANIZATION] [TEAM]",
```

Comment:
This should list for all the resources or else just the specified resource the teams the user belongs to that has access to that resource as well as the level of access.